ui/widget.js

```diff
@@ -12,6 +12,7 @@
 //>>description: Provides a factory for creating stateful widgets with a common API.
 //>>docs: http://api.jqueryui.com/jQuery.widget/
 //>>demos: http://jqueryui.com/widget/
+//>>css.theme: ../themes/base/theme.css
```

Comment:
No this should not be here only things that actually depend on css should have a dependency on it . Individual widgets should depend on theme that actually use it its very very possible to have widgets that require no css at all ( though not common ). So this would include unnecessary files if someone was for example just trying to do a minimal build of `core.js` and `widget.js` for doing custom widget development or something.